### PR TITLE
Release alpha02

### PR DESCRIPTION
**Pull request by zsoltk**
## Check list

- [x] I have updated `CHANGELOG.md`
- [x] I don't need to update documentation

### CODE REVIEW

**Review by KovalevAndrey — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,14 +1,19 @@
-# Change Log
+# Changelog
```
Comment (LachlanMcKee) [inline CHANGELOG.md:1]: Thoughts on adding the changelog to the website? (can always be done in another PR)

Comment (zsoltk) [reply CHANGELOG.md:1]: Good point! I'll do separately as I have other changes to docs anyway.
